charts/images.yaml

```diff
@@ -68,32 +64,56 @@ images:
 - name: csi-provisioner
   sourceRepository: github.com/kubernetes-csi/external-provisioner
   repository: k8s.gcr.io/sig-storage/csi-provisioner
-  tag: "v2.1.2"
+  tag: "v2.2.2"
+  targetVersion: "< 1.20"
+- name: csi-provisioner
+  sourceRepository: github.com/kubernetes-csi/external-provisioner
+  repository: k8s.gcr.io/sig-storage/csi-provisioner
+  tag: "v3.1.0"
+  targetVersion: ">= 1.20"
 - name: csi-attacher
   sourceRepository: github.com/kubernetes-csi/external-attacher
   repository: k8s.gcr.io/sig-storage/csi-attacher
-  tag: "v3.3.0"
+  tag: "v3.4.0"
+- name: csi-resizer
+  sourceRepository: github.com/kubernetes-csi/external-resizer
+  repository: k8s.gcr.io/sig-storage/csi-resizer
+  tag: "v1.4.0"
 - name: csi-snapshotter
   sourceRepository: github.com/kubernetes-csi/external-snapshotter
   repository: k8s.gcr.io/sig-storage/csi-snapshotter
   tag: "v3.0.3"
+  targetVersion: "< 1.20"
+- name: csi-snapshotter
+  sourceRepository: github.com/kubernetes-csi/external-snapshotter
+  repository: k8s.gcr.io/sig-storage/csi-snapshotter
+  tag: "v5.0.1"
```

Comment:
Shouldn't the external-snapshotter components run the same version. Now csi-snapshotter sidecar is at v5.0.1, the napshot-validation-webhook is at v4.1.1 and the snapshot-controller is at v4.2.1. Can you elaborate why?

Reply:
I took the version from here https://kubernetes-csi.github.io/docs/snapshot-controller.html. I will update all of them to v6.0.1

Reply:
Hm, I didn't checked the changelogs yet but it sounds scary that we jump from v3.0.3 to v6.0.1. Are we sure that there are no breaking changes in-between?

Reply:
I tested it in past it was working fine but for the safe side, I will for now update to the latest minor version of v4? Will that be fine?

Reply:
Yep, sounds safer.